.github/workflows/industrial_ci_action.yml

```
@@ -0,0 +1,105 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: BuildAndTest
+
+# this is a test change, affects no projects
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+      - "[kmn]*-devel"
+
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - IMAGE: 'master-ci'
+            CCOV: true
+            AFTER_RUN_TARGET_TEST: './.ci.prepare_codecov'
+          - IMAGE: 'master-ci-shadow-fixed'
+            CATKIN_LINT: true
+          - IMAGE: 'melodic-ci'
+            BEFORE_TARGET_TEST_EMBED: 'set +u && source moveit_kinematics/test/test_ikfast_plugins.sh && set -u'
+            CLANG_TIDY: true
+    env:
+      DOCKER_IMAGE: moveit/moveit:${{ matrix.env.IMAGE }}
+      UPSTREAM_WORKSPACE: moveit.rosinstall
+      CCOV: ${{ matrix.env.CCOV }}
+      AFTER_SETUP_UPSTREAM_WORKSPACE: 'vcs pull $BASEDIR/upstream_ws/src'
```

reorder to group semantically similar things:
```suggestion
      UPSTREAM_WORKSPACE: moveit.rosinstall
      AFTER_SETUP_UPSTREAM_WORKSPACE: 'vcs pull $BASEDIR/upstream_ws/src'
      CCOV: ${{ matrix.env.CCOV }}
```